mudlet-mapper.xml

```diff
@@ -4685,7 +4685,7 @@ mmp.echo(&quot;All astral nodes have been &quot;..(matches[2]:lower()==&quot;on&
 -- Note that you can also use external Scripts --
 -------------------------------------------------
 
-function createOption(startingValue, onChangeFunc, allowedVarTypes, use, checkOption)
+function createOption(startingValue, onChangeFunc, allowedVarTypes, use, games, checkOption)
```

Comment:
Adding the parameter not as the last one breaks backwards compatibility, I fear. If people created private options and used the `checkOption` parameter, their scripts will break.

I'm aware that this means a number of ugly `nils` in the creation of the options below.